jda/src/main/java/co/aikar/commands/JDAOptions.java

```diff
@@ -6,7 +6,7 @@
 public class JDAOptions {
     CommandConfig defaultConfig = new JDACommandConfig();
     CommandConfigProvider configProvider = null;
-    CommandPermissionResolver permissionResolver = new JDACommandPermissionResolver();
```

Comment:
per last comment, we just need to pass the manager to the method parameter, so you can revert this back.